src/taipy/core/_scheduler/_dispatcher/_job_dispatcher.py

```diff
@@ -138,7 +137,6 @@ def _run_wrapped_function(cls, mode, config_as_string, job_id: JobId, task: Task
         try:
             if mode != JobConfig._DEVELOPMENT_MODE:
                 Config._applied_config = _TomlSerializer()._deserialize(config_as_string)
-                Config.block_update()
```

Comment:
I don't understand this change.
This is executed in a subprocess. We build a new singleton config. then we want to block the config update. No? Otherwise we do not protect a task from updating the config.

Reply:
Oh yes. I forgot to block config update on subprocess. My bad.

I moved the `Config.block_update()` to the `Core().run()` method so that it will be consistent with the `Core().stop()` method.
However, it might not be necessary.